SCClassLibrary/Common/GUI/Base/Menu.sc

```diff
@@ -106,7 +106,9 @@ MainMenu {
 	classvar systemMenus;
 	classvar <>buildAppMenusAction;
 
-	*initClass {
+	*initClass {}
+
+	*initControlMenuActions {
```

Comment:
thanks! this name is technically accurate but feels a little awkward to me -- maybe `initBuiltInMenus` instead?